app/models/user.rb

```diff
@@ -146,11 +147,11 @@ def office_head?
   end
 
   def has_admin_access?
-    admin? || open_source_manager? || hr?
+    admin?
   end
 
   def is_admin?
-    admin? || hr?
+    admin?
   end
```

Comment:
can we change this to a single method now? or change it to an alias if you prefer

Reply:
I was thinking about it, but not sure if we gonna need this with another kind of role, maybe for the evaluator kind? @adbatista ?

Also, seems that this is to check if the user can access the active admin, and some elements in the dashboard, but also it does not makes sense. Maybe in another PR.